users/views.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework             import serializers
+from rest_framework.generics    import CreateAPIView
+from rest_framework.permissions import AllowAny
+
+from .serializers import RegisterUserSerializer
+
+
+class RegisterUsertView(CreateAPIView):
+    permission_classes = [AllowAny]
+    serializer_class = RegisterUserSerializer
```

Comment:
여기도 정렬하면 보기 좋을 것 같습니다!
```suggestion
    permission_classes = [AllowAny]
    serializer_class   = RegisterUserSerializer
```

Reply:
수정완료입니다!